packages/neuron-ui/src/components/HistoryDetailPage/index.tsx

```diff
@@ -202,9 +240,14 @@ const HistoryDetailPage = () => {
       dataIndex: 'amount',
       align: 'left',
       isBalance: true,
+      className: styles.amount,
       render(_, __, item, show: boolean) {
         const { capacity } = handleListData(item)
-        return show ? <CopyZone content={capacity.replace(/,/g, '')}>{`${capacity} CKB`}</CopyZone> : HIDE_BALANCE
+        return show ? (
+          <CopyZone maskRadius={8} content={capacity.replace(/,/g, '')}>{`${capacity} CKB`}</CopyZone>
```

Comment:
`capacity.replaceAll(',', '')` can be used now

Ref:
  - Compatibility: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll#browser_compatibility
  - Chrome in Electron: https://www.electronjs.org/blog/electron-24-0#notable-changes